projects/netgrif-components/src/lib/panel/case-panel/case-panel.component.ts

```diff
@@ -15,7 +16,7 @@ export class CasePanelComponent extends AbstractCasePanelComponent {
                 protected _snackBarService: SnackBarService, protected _translateService: TranslateService,
                 protected _log: LoggerService, @Optional() protected _overflowService: OverflowService,
                 protected _userService: UserService, protected _currencyPipe: CurrencyPipe,
-                public _permissionService: PermissionService) {
+                public _permissionService: PermissionService, protected _activatedRoute: ActivatedRoute) {
```

Comment:
why is the activated route added as an attribute if it is not referenced anywhere?

Reply:
Corrected.